src/common/components/nav/navCart/navCart.component.js

```diff
@@ -43,7 +43,9 @@ class NavCartController{
           this.loading = false;
           this.hasItems = !isEmpty(this.cartData.items);
           this.analyticsFactory.buildProductVar(data);
-          this.analyticsFactory.setEvent('cart open');
+          if(this.cartData.items.length === 1){
+            this.analyticsFactory.setEvent('cart open');
```

Comment:
This will get fired if the cart has 2 items and one is deleted and if the cart has one item and it is edited. Also it will get triggered on a page refresh/navigation.

Is the intent of this event to only capture when a user first starts using their cart/starts an order?

There's a possibility adding this event here will work: https://github.com/CruGlobal/give-web/blob/master/src/common/services/api/cart.service.js#L106-L106. We're starting a new session for guest users. We'd have to figure out how to handle non guest users if we want to fire the event here.